build-system/server.js

```diff
@@ -33,6 +33,14 @@ var url = require('url');
 app.use(bodyParser.json());
 app.use(morgan('dev'));
 
+app.use('/', function(req, res, next) {
+  res.setHeader('AMP-Access-Control-Allow-Source-Origin',
```

Comment:
I'd love to be more specific.  I just wasn't sure how to detect 'this is the AMP runtime making an XHR request to a localhost resource.'  I could code it to the path where we're storing the data files for this fake impl, but I worried that that might be *overly* restrictive.  I'd really just like to detect the XHR-to-local case in general.  Do you know a way to do that?  Or have a preferred way of handling it?

Thanks!